### ISSUE TITLE
Se você programa em Windows ao invés de Linux não deve ser um bom dev

### ISSUE THREAD

**Issue by allangrds** [ospilhacheia/tretadev#26]

**Comment by alysson3dev** [comment]
O senhor está deverás certo. meu amigo

**Comment by fabianocouto** [comment]
Não seja idiota! O que tem a ver a habilidade de um profissional com o sistema operacional que ele usa? Tem um monte de gente fazendo um monte de besteira em tudo que é sistema operacional, ide, linguagem, etc... Assim como tem muita gente muito boa também.

**Comment by allangrds** [comment]
@fabianocouto não entendi a agressividade. A ideia das issues aqui não é colocar temas bobos de devss?

**Comment by allangrds** [comment]
@fabianocouto não entendi a agressividade. A ideia das issues aqui não é colocar temas bobos de devs?

cc @lincolixavier

**Comment by alysson3dev** [comment]
Até o momento pensei que a ideia por trás do repo era apenas pra entreter e não realmente ter treta.

**Comment by allangrds** [comment]
"Repositório criado com o intuito de centralizar as tretas semanais que acontecem na cena dev BR" - acho que compensa mudar e deixa ainda mais explícito o objetivo desse repo.

**Comment by fabianocouto** [comment]
Ué! hauahuaa... Eu pensei que tinha entendido, mas agora eu fiquei confuso. Então vocês querem treta sem treta? Devo apagar o comentário? Eu só resolvi entrar na brincadeira, mas acho que não entendi mesmo, então.

**Comment by allangrds** [comment]
Se vc entrou na brincadeira, blz, hahahah. Vou remover os meus comentários, então. Deixa o seu, mas só coloca que é brinks xD

**Comment by fabianocouto** [comment]
:) Blz! Claro que é brinks. Só queria fazer uma observação, eu já não Windows a mais de 15 anos. Mas isso não quer dizer nada, obviamente. hauhauah

**Comment by alysson3dev** [comment]
Carai @fabianocouto Hollywood deve te contratar pqp. Q atuação. não há pessoa q fale q isso é um comentário irônico

**Comment by lincolixavier** [comment]
Chamam de Ruindowns a toa?

**Comment by fabianocouto** [comment]
> Carai @fabianocouto Hollywood deve te contratar pqp. Q atuação. não há pessoa q fale q isso é um comentário irônico

Isso é porque você nunca conversou comigo pessoalmente. Mas eu falei sério. É treta real.

**Comment by fabianocouto** [comment]
> Chamam de Ruindowns a toa?

Sim. Isso é puro preconceito. Não faz sentido algum, o sistema operacional com o melhor tratamento de erros, receber esse rótulo. Sem contar que é berço do melhor navegador da história, o Internet Explorer. Se tem uma coisa que eu sinto falta no linux, é do Internet Explorer.

**Comment by alysson3dev** [comment]
> > Chamam de Ruindowns a toa?
> 
> Sim. Isso é puro preconceito. Não faz sentido algum, o sistema operacional com o melhor tratamento de erros, receber esse rótulo. Sem contar que é berço do melhor navegador da história, o Internet Explorer. Se tem uma coisa que eu sinto falta no linux, é do Internet Explorer.

Vdd. Nunca q vc vai ficar preso numa tela de atualização inesperada que demora 3 horas pra ficar pronto😊

**Comment by lincolixavier** [comment]
Esse povo inventa cada coisa

**Comment by jmurowaniecki** [comment]
> > > Chamam de Ruindowns a toa?
> > 
> > 
> > Sim. Isso é puro preconceito. Não faz sentido algum, o sistema operacional com o melhor tratamento de erros, receber esse rótulo. Sem contar que é berço do melhor navegador da história, o Internet Explorer. Se tem uma coisa que eu sinto falta no linux, é do Internet Explorer.
> 
> Vdd. Nunca q vc vai ficar preso numa tela de atualização inesperada que demora 3 horas pra ficar pronto

Melhor uma tela azul e uma reinicialização eventual do que um problema com driver na placa de vídeo e não poder jogar no LOLzinho no Linux `pacman -Syyu` não difere em nada do Windows Update\</desabafo>

**Comment by jdoper** [comment]
A galera tá muito ultrapassada, to super de boas programando em Java no meu PC da xuxa. LINUX E WINDOWS É COISA DO PASSADO!!!

**Comment by terremoth** [comment]
Essa é realmente uma frase sem sentido. Programar é usar ferramenta (linguagens de programação são como ferramentas) pra construir coisas. É como usar um machado ou martelo...

No fim é, esse título da issue é como dizer: saber usar uma ferramenta na casa do João é melhor do que na casa do Pedro: não faz sentido (desde que a ferramenta estejam disponíveis em ambas as casas).

O que difere um dev bom de um ruim, ao meu ver, é uma série de coisas:
1. análise sistêmica dos processos do software que quer desenvolver;
2. conhecer bem as ferramentas (linguagens/IDE's etc) que está usando para desenvolver;
3. saber dialogar e formular perguntas bem feitas;
4. saber pesquisar direito pra encontrar exatamente a resposta pra aquilo que procura;
5. ter hábito de ler e pesquisar nas documentações das tecnologias usadas/aprendidas;
6. não ser refém de poucos cenários;
7. saber fazer testes e debugar;
8. saber aplicar princípios SOLID e design patterns quando necessários;
9. saber mais de uma linguagem de programação e saber trabalhar com pelo menos 1 banco de dados;
10. saber escrever códigos manuteníveis e legíveis, para que outros possam dar continuidade a seu trabalho;
11. procurar aprender sempre, nossa área vive mudando e adicionando/removendo coisas/informações

Deve ter mais coisas, mas só consegui pensar nessas 11 kkkk
mals aí pelo texto palestrinha

**Comment by GuilhermeLimaRepository** [comment]
Nem se for Dev C# ?? Kkkkkk
